File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
   let count = 0;
   const bables = ["aya","ye","woo","ma"];
   for(const [idx, str] of babbling.entries()) {
       for(const str2 of bables){
          babbling[idx] = babbling[idx].replaceAll(str2, " ");
       }
   }
   for(let i = 0; i < babbling.length; i++) {
       if(babbling[i].trim().length == 0) {
           count++;
       }
   }
    
   return count;
   
}

File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.java
import java.lang.*;
import java.util.*;
class Solution {
    public int solution(String[] babbling) {
        List<String> bables = new ArrayList<>();
        List<String> list = new ArrayList<>();
        bables.add("aya");
        bables.add("ye");
        bables.add("ma");
        bables.add("woo");
        int answer = 0;
        for(int i = 0 ; i < babbling.length; i++){
           for(String str2: bables){
               if(babbling[i].contains(str2)) {
                  babbling[i] = babbling[i].replace(str2, " ");
               }
           }
        }
        for(String str: babbling) {
            if(str.trim().isEmpty()) {
                answer++;
            }
        }
        return answer;
    }
}